### PR TITLE
Add affinity labels UI feature page

### PR DESCRIPTION
**Pull request by pcbailey**
Changes proposed in this pull request:

- Add a feature page for the addition of the affinity labels webadmin UI

- Update the GUI section of the affinity labels documentation page to
point to the new feature page

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @pcbailey

This pull request needs review by: @MarSik @doron-fediuck @yanirq @jtokar @akrejcir @akashihi

### CODE REVIEW

File: source/develop/release-management/features/sla/affinity-labels-management-via-webadmin.html.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+*   Owner: Phillip Bailey
+*   Email: phbailey@redhat.com
+
+## Detailed Description
+
+[Affinity labels](/documentation/sla/affinity-labels) were introduced in version 4.0, but could only be managed via the REST API. This feature allows users to manage their labels without ever leaving the webadmin UI.
+
+### Affinity Label Sub Tabs
+
+A sub tab has been added to the clusters, virtual machines, and hosts main tabs. The sub tabs for virtual machines and hosts display the labels that have been assigned to the currently selected entity. For the moment, the clusters sub tab displays all labels in the system. In the future, it is intended to restrict labels at the cluster level. At that time, the clusters sub tab will only display labels associated with the selected cluster.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/sla/affinity-labels-management-via-webadmin.html.md]: I am not 100% sure that the description of the location of the affinity tabs is clear.

**Comment by mykaul** [comment]
@pcbailey - what's the status of this one?

File: source/develop/release-management/features/sla/affinity-labels-management-via-webadmin.html.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+title: Affinity Labels Management via Webadmin
+category: feature
+authors: phbailey
+wiki_category: SLA
+feature_name: Affinity Labels Management via Webadmin
+feature_modules: Webadmin
+feature_status: Complete
+---
+
+# Affinity Labels Management via Webadmin
```
[the remark this answers is not in the document]
Comment (pcbailey) [reply source/develop/release-management/features/sla/affinity-labels-management-via-webadmin.html.md]: Done.

File: source/develop/release-management/features/sla/affinity-labels-management-via-webadmin.html.md (hunk 1 above)
Comment (pcbailey) [reply source/develop/release-management/features/sla/affinity-labels-management-via-webadmin.html.md]: The sub tabs are different from the side tabs in the popups, but I incorporated your advice for those in the section that talks about the side tabs. What used to be the sub tabs are now the detail tabs in 4.2. I updated the language to the 4.2 nomenclature and created bulleted lists for the steps to make it easier to follow.

**Comment by pcbailey** [comment]
@mykaul - I'm not sure how I missed John's feedback. I have updated the PR to incorporate his suggestions and use the 4.2 UI nomenclature.

**Pull request merged**